woocommerce/wc-api-dev
# Store: Try an endpoint for updating certain TaxJar settings from Calypso.

As a follow-up to my comment here: https://github.com/Automattic/wp-calypso/issues/17110#issuecomment-334311749 - I wanted to try the idea of adding a Calypso specific endpoint (i.e. one not destined for core) for managing the TaxJar settings.

As I mentioned in that comment, integrations are not currently exposed via the REST API. I talked about just deactivating the plugin if the user toggles tax calculations off, but I worry that could complicate possibly moving the plugin to wpcomsh later. I would rather update the actual setting.

These endpoints (including things like the upcoming Mailchimp endpoints) are also candidates to move to a Calypso compatibility plugin along with the hotfixes -- so I added some TODOs around that.

To Test:
* Run `phpunit` and make sure all tests pass.
* You can make requests to `/wc/v3/calypso/taxjar-settings` using something like Postman, and verify that you can see the `enabled` setting, and that you can update it to true/or false.
